js/scripts.js

```diff
@@ -0,0 +1,16 @@
+var heightOfTree = prompt('Enter height of tree');
+
+function drawTree(heightOfTree) {
```

Comment:
zdajesz sobie sprawę, żę zmienna `heightOfTree` nie musi być taka sama jak ta z pierwszej linijki? Jeśli tak to git.

Reply:
W sumie to numer mi nie działał jako parametr drawTree więc umieściłem tam tę zmienną.